### ISSUE TITLE
Suggestions on how to add custom snippets

### ISSUE THREAD

**Issue by LennDG** [zefoy/ngx-ace-wrapper#5]
I am trying to add a custom snippet file to the ace editor. Are there any suggestions how to do that through this angular wrapper for ace?

The basic snippet files included in brace are not enough and I need to extend them.

**Comment by sconix** [comment]
I think ACE has API for adding snippets so perhaps ask in ACE related forums. This wrapper does not offer any special API for it so its all about ACE and its API.

**Comment by LennDG** [comment]
I have looked into the ACE API but it does not have any API calls available for adding snippets. I did find something about getting the snippetmanager from ace but that means I need to get the ace instance. Does this wrapper expose the ace instance it uses anywhere?

**Comment by sconix** [comment]
Yes its exposed, just see the README or example app how to get it.
